### DISCUSSION TITLE
erreur d'installation de security onion

### DISCUSSION THREAD

**Discussion by Oumar79** [Security-Onion-Solutions/securityonion#13432] [2.4]
### Version

2.4.80

### Installation Method

Security Onion ISO image

### Description

installation

### Installation Type

Eval

### Location

on-prem with Internet access

### Hardware Specs

Meets minimum requirements

### CPU

4

### RAM

16

### Storage for /

80

### Storage for /nsm

80

### Network Traffic Collection

span port

### Network Traffic Speeds

Less than 1Gbps

### Status

No, one or more services are failed (please provide detail below)

### Salt Status

No, there are no failures

### Logs

No, there are no additional clues

### Detail

j'ai un message d'erreur lorsque j'essaye d'installer security onion Eval 

### Guidelines

- [X] I have read the discussion guidelines at https://github.com/Security-Onion-Solutions/securityonion/discussions/1720 and assert that I have followed the guidelines.

**Comment by Oumar79** [comment]
![blob](https://github.com/user-attachments/assets/28492d27-9daf-4dbf-9033-3f8fbdf46a36)
voici l'erreur en question

**Reply by cm-ops** [reply]
Have you looked at those logs listed? What are the errors you see?

**Reply by Oumar79** [reply]
![WhatsApp Image 2024-08-05 à 14 45 59_5b660bff](https://github.com/user-attachments/assets/a05ab563-bb56-47f1-a993-eb6ef40f1439)
voici le fichier log

**Reply by cm-ops** [reply]
How about the `/root/sosetup.log`, that will have more context around those errors.
